buildrpm/python-drgn-tools.spec

```diff
@@ -116,6 +117,7 @@ rm %{buildroot}/usr/bin/DRGN
 %{python3_sitelib}/drgn_tools/*
 /usr/bin/corelens
 %{_mandir}/man1/corelens.1.gz
+%{buildroot}%{python3_sitelib}/sos/report/plugins/
```

Comment:
This indicates that this RPM owns the directory and all its contents. Really we only want to own the file. Can you change this to specify the specific file?